### ISSUE TITLE
bug: Shovel not updating coverimages

### ISSUE THREAD

**Issue by camrun91** [ApollosProject/apollos-apps#2136]
@vinnyjth @conrad-vanl It looks like in a recent update to the shovel the cover image diag is no longer mapping the content items so it is never setting the cover image ID. 

PS I am opening this here because issues are not open in the shovel repo.

**Comment by vinnyjth** [comment]
Can you clarify what you mean by "no longer mapping the content items"? 

Are all items missing cover image ids, or are only some? 

Do the items missing cover image ids also have no set parent ids? 

We know this is working for core and our church apps, but there might be a new config option or change that is breaking things for you.

**Comment by camrun91** [comment]
Basically, I found that all content items had no cover image id's in digging I found that in a recent update this line `content_items = list(map(self.map_content_items, rock_objects))` got removed from this location https://github.com/ApollosProject/apollos-shovel/blob/aa9dd8be794ea0aebde601c02aade83c8da1a52d/dags/rock/rock_cover_image.py#L132

so with that line missing the `map_content_items` function never got called. I added that line back and things now have cover images as expected. 

If there is another way to trigger that now I am more than happy to switch to that way.
